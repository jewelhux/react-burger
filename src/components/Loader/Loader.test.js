import React from 'react';
import renderer from 'react-test-renderer';
import Loader from './Loader';

it('Ссылка рендерится без ошибок', () => {
  const tree = renderer.create(<Loader />).toJSON();
  expect(tree).toMatchSnapshot();
});
