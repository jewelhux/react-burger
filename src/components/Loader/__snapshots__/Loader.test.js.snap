// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ссылка рендерится без ошибок 1`] = `
<div
  className="loader"
>
  <div
    className="spinner"
  />
</div>
`;
